package.json

```diff
@@ -4,6 +4,9 @@
   "private": true,
   "scripts": {
     "build": "lessc src/styles.less src/styles.css && node scripts/build.js",
+    "build:test": "env-cmd -e test -r .cmdrc.json npm run build",
```

Comment:
the test environment for spruce is not clear to me yet. if using a mock server then that should be the the test server. do we have plans to point spruce to a local evergreen server?

Reply:
oh i see what you're saying. i meant to delete this command since no configuration currently exists to run a local evg instance alongside spruce. can add the test env that points to local evergreen instance

Reply:
We can. Actually I think if you set REACT_APP_UI_URL to localhost:9090 and REACT_APP_API_URL to localhost:9090/api, I think it will just work assuming you've run `make smoke-start-server`

Reply:
the `env-cmd` is for build environments. to set env variables in dev it's just set through shell like `REACT_APP_UI_URL=localhost:9090 npm start`. i added a start script that does this.